### PR TITLE
fix: move acronym data to typescript file

### PR DESCRIPTION
**Pull request by dblandin**
Minor fix to bring the acronym data into typescript.

Seemed easier than keeping it as JSON and figuring out the npm packaging change there.

### CODE REVIEW

**Comment by anandaroop** [comment]
Nice!

**Comment by artsyit** [comment]
:rocket: PR was released in [`v0.6.1`](https://github.com/artsy/artsy-cli/releases/tag/v0.6.1) :rocket:

File: src/commands/acronym.ts [hunk 1]
```diff
@@ -15,16 +15,16 @@ AOV: Average Order Value`,
...
     const { args } = this.parse(Acronym)
-    const result = findAcronym(args.query)
+    const result = findAcronym(args.acronym)
 
     if (result) {
-      console.log(result)
+      this.log(result)
     } else {
-      console.log("Sorry, unknown acronym")
+      this.log("Sorry, unknown acronym")
```
Comment (anandaroop) [inline src/commands/acronym.ts:27]: Ah right, forgot about this

**Comment by artsyit** [comment]
:rocket: PR was released in [`v0.6.1`](https://github.com/artsy/artsy-cli/releases/tag/v0.6.1) :rocket:

**Comment by artsyit** [comment]
:rocket: PR was released in [`v0.6.1`](https://github.com/artsy/artsy-cli/releases/tag/v0.6.1) :rocket:

**Comment by artsyit** [comment]
:rocket: PR was released in [`v0.6.1`](https://github.com/artsy/artsy-cli/releases/tag/v0.6.1) :rocket:

File: src/commands/acronym.ts [hunk 2]
```diff
@@ -15,16 +15,16 @@ AOV: Average Order Value`,
     help: flags.help({ char: "h" }),
   }
 
-  static args = [{ name: "query" }]
+  static args = [{ name: "acronym", required: true }]
```
Comment (anandaroop) [inline src/commands/acronym.ts:18]: 👍

**Review by anandaroop — COMMENTED** [review]
Thanks for the comments and updates 💯

**Pull request merged**